libs/zedUpload/httputil/http.go

```diff
@@ -18,6 +18,7 @@ import (
 	"strings"
 	"time"
 
+	logutils "github.com/lf-edge/eve/pkg/pillar/utils/logging"
```

Comment:
Seems it requires go.mod update to include `github.com/lf-edge/eve/pkg/pillar`. If I correctly understood, this library is used by cloud side.

Reply:
i did 'make proto-vendor'. do you see any compiling issue @giggsoff ?

Reply:
I can see problems with `go get github.com/lf-edge/eve/libs/zedUpload`:
```
...
go get: github.com/lf-edge/eve/pkg/pillar@v0.0.0-20210922101739-3840a435ad43 requires
        github.com/lf-edge/eve/api/go@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

Seems, it comes with our dummy version in pillar/go.mod.